collateral/deployment/k8s-without-helm-reference.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Sourcegraph to Kubernetes (k8s) without Helm
+
+## Introduction
+This material is intended to be used as a handout to customers working with the Customer Engineering team. The materials are meant to serve as an overview and resource to aid in deep diving into Sourcegraph's documentation and common deployment tasks.
+
+**Table of Contents:**
+
+- [Introduction](#introduction)
+- [Sourcegraph Setup and Deployment](#sourcegraph-setup-and-deployment)
+	- [Prerequisites](#prerequisites)
+  - [Create your cluster](#create-your-cluster)
+  - [Configure your manifest file](#configure-your-manifest-file)
+  - [Install Sourcegraph](#install-sourcegraph)
+- [Next Steps](#next-steps)
+
+## Sourcegraph Setup and Deployment
+
+### Prerequisites
+- Ensure you have your Sourcegraph Enterprise License
+- Access to deploy Kubernetes clusters
+  - Minimum Kubernetes version: `v1.19` and `kubectl` `v1.19`
+  - Support for Persistent Volumes (SSDs recommended)
+- Determine resource requirements to properly size your instance: [Resource Estimator](https://docs.sourcegraph.com/admin/deploy/resource_estimator)
+- Install relevant CLIs for your provider of choice:
+  - AWS: [`aws` cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [`ejsctk`](https://eksctl.io/introduction/#installation)
+  - Azure: [`az` cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  - GCP: [`gcloud` cli](https://cloud.google.com/sdk/gcloud)
+
+### Create your cluster
+- Follow the instructions for your provider of choice to create your Kubernetes (k8s) cluster: [Cloud installation instructions](https://docs.sourcegraph.com/admin/deploy/kubernetes#cloud-installation)
+  - Follow the recommendations for `node type`, `disk type`, and `minimum` and `maximum` nodes
+
+### Configure your manifest file
+Sourcegraph has a list of configurations available and recommended that need to be made to your manifest file to properly install Sourcegraph into your cluster. Please review the full configuration guide prior to proceeding: [Configure Sourcegraph with Kubernetes](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure)
```

Comment:
I worry that this is just pointing them to a very long doc without much guidance.

Reply:
@emchap the goal of this being a reference to a single source of truth, which should be our documentation, tells me that if that doc isn't good enough the team in charge of it needs to improve that doc.

I don't disagree, there's a LOT of information on that page about editing your manifest, but I wholeheartedly think this should be a reference to official documentation and documentation should be updated by the team that owns that to ensure it's correct, accurate, and usable. A single source of truth that is our official documentation makes way more sense in terms of supporting customers and long-term scalability.

@amenne I'm curious about your thoughts on this since we chatted a bit about docs this week too and content in the docs versus navigation of the docs.

Reply:
```suggestion
Sourcegraph has a list of configurations available and recommended that need to be made to your manifest file to properly install Sourcegraph into your cluster. This includes how to customize the deployment using Kustomize—if you prefer to use Helm, please see the [Helm install instructions](https://docs.sourcegraph.com/admin/deploy/kubernetes/helm), instead. 

Please review the full configuration guide prior to proceeding: [Configure Sourcegraph with Kubernetes](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure). Pay particular attention to configuring a storage class for your cloud provider, configuring network access, and configuring external databases if you choose to use them.

Other sections of the configuration page should be reviewed, but are not required (for example, an external Jaeger instance is optional).
```